CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Minor
 
 - Button: Fix transparent button on blue background (#316)
+- Flyout: Add new prop `shouldFocus` to override focus on open behavior. *Has codemod* (#324)
```

Comment:
@jennyscript could you change this to `#325`?

https://github.com/pinterest/gestalt/pull/324 leads to an enzyme upgrade